microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -88,7 +88,10 @@ The following table shows all available hints request headers with sample values
 You can access User-Agent Client Hints using JavaScript on the client side. When you call the default `navigator.userAgentData`, it returns the following response.
 
 ```JSON
-{ brands: [ {brand: "Chromium","version":"91"}, {brand: "Microsoft Edge","version":"91"}, {brand: "GREASE","version":"99"}, ]
+{ brands:
+    [ {brand: "Chromium","version":"91"},   
+    {brand: "Microsoft Edge","version":"91"},  
+    {brand: "GREASE","version":"99"} ]  
 mobile: false }

```

Comment:
If we want to reformat this to be easier to read, we should probably format like this instead:

```json
{
  "brands": [
    {
      "brand": "Chromium",
      "version":"91"
    },
    {
      "brand": "Microsoft Edge",
      "version":"91"
    },
    {
      "brand": "GREASE",
      "version":"99"
    }
  ],
  "mobile": false 
}
```

Reply:
done